### ISSUE TITLE
Activation Functions - GELU?

### ISSUE THREAD

**Issue by leftwinglow** [leffff/graphormer-pyg#23]
Hi @leffff , I've been playing around with your model for a few days but I haven't been able to find the activation functions.

I believe the original paper uses GELU.

If you could let me know where they are I'd appreciate it.

Thanks,

Luke.

**Comment by leffff** [comment]
Hey, Luke!
Well I have indeed made a mistake and initialized FFN as Linear. It should be replaced by 
```python3
 nn.Sequential(
    nn.Linear(hiden_dim, ffn_dim),
    nn.ReLU(),
    nn.Dropout(p),
    nn.Linear(ffn_dim, hiden_dim)

)
```

**Comment by leffff** [comment]
Oh sorry GELU

**Comment by leffff** [comment]
fixed: 231fcf0efcb809b04d20f7a9272ffc10ee41fd83
